pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -609,6 +609,12 @@
      */
     ConsumerBuilder<T> enableRetry(boolean retryEnable);
 
+    /**
+     * Enable or disable the batch index acknowledgment. To enable this feature must ensure batch index acknowledgment
+     * feature is enabled at the broker side.
+     */
+    ConsumerBuilder<T> batchIndexAcknowledgmentEnabled(boolean batchIndexAcknowledgmentEnabled);
```

Comment:
Shouldn't it be called **enableBatchIndexAcknowledgement** to conform other settings?

Reply:
Ok